nilearn/_utils/docs.py

```diff
@@ -15,45 +15,50 @@
 #
 docdict = dict()
 
+NILEARN_LINKS = {"landing_page": "http://nilearn.github.io"}
+NILEARN_LINKS["input_output"] = "{}/manipulating_images/input_output.html".format(
+    NILEARN_LINKS["landing_page"]
+)
+
 # Verbose
 verbose = """
-verbose : int, optional
+verbose : :obj:`int`, optional
     Verbosity level (0 means no message).
     Default={}."""
 docdict['verbose'] = verbose.format(1)
 docdict['verbose0'] = verbose.format(0)
 
 # Resume
 docdict['resume'] = """
-resume : bool, optional
+resume : :obj:`bool`, optional
     Whether to resumed download of a partly-downloaded file.
     Default=True."""
 
 # Data_dir
 docdict['data_dir'] = """
-data_dir : string, optional
+data_dir : :obj:`str`, optional
     Path where data should be downloaded. By default,
     files are downloaded in home directory."""
 
 # URL
 docdict['url'] = """
-url : string, optional
+url : :obj:`str`, optional
     URL of file to download.
     Override download URL. Used for test only (or if you
     setup a mirror of the data).
     Default=None."""
 
 # Smoothing_fwhm
 docdict['smoothing_fwhm'] = """
-smoothing_fwhm : float, optional.
-    If smoothing_fwhm is not None, it gives the size in millimeters of the
+smoothing_fwhm : :obj:`float`, optional.
+    If `smoothing_fwhm` is not None, it gives the size in millimeters of the
```

Comment:
I think single backticks render oddly with Sphinx, so I generally use double backticks, which render as code snippets.

Reply:
Thanks for the review! I changed the single backticks into double in the standard entries.